fiftyone/utils/moge.py

```
@@ -0,0 +1,383 @@
+
+    def _load_model(self, config):
+        model = moge_v3.MoGeModel.from_pretrained(config.name_or_path)
+        model = model.to(self._device)
+        model.eval()
+        return model
+
+    @property
+    def media_type(self):
+        return "image"
+
+    def _forward_pass(self, imgs):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_forward_pass`

(ANN202)